fortytwo.cabal

```diff
@@ -35,8 +35,8 @@ library
                        FortyTwo.Renderers.Select
                        FortyTwo.Renderers.Multiselect
   build-depends:       base >= 4.7 && < 5,
-                       text <= 1.2.2.2,
-                       ansi-terminal >= 0.6.0.0 && <= 0.7.1.1
+                       text,
```

Comment:
Isn't that dangerous avoid completely the version boundaries?

Reply:
oh good catch, that is a mistake :D will fix it